adagrams/game.py

```diff
@@ -1,11 +1,146 @@
+import random 
+import string
+import copy
+
+
+LETTER_POOL = {
+    'A': 9, 
+    'B': 2, 
+    'C': 2, 
+    'D': 4, 
+    'E': 12, 
+    'F': 2, 
+    'G': 3, 
+    'H': 2, 
+    'I': 9, 
+    'J': 1, 
+    'K': 1, 
+    'L': 4, 
+    'M': 2, 
+    'N': 6, 
+    'O': 8, 
+    'P': 2, 
+    'Q': 1, 
+    'R': 6, 
+    'S': 4, 
+    'T': 6, 
+    'U': 4, 
+    'V': 2, 
+    'W': 2, 
+    'X': 1, 
+    'Y': 2, 
+    'Z': 1
+}
+
 def draw_letters():
-    pass
+    
+    letters_list = []
+
+    for char, frequency in LETTER_POOL.items():
+        letters_list.extend([char] * frequency)
+
+
+    hand = []
+    for i in range(10):
+        randomLetter = random.choice(letters_list)
+        hand.append(randomLetter)
+        letters_list.remove(randomLetter)
+    
+
+    #*********
+    #YET TO SEE IF FREQUENCY IS LESS THAN THE LETTER POOL
+    return hand
+    
 
 def uses_available_letters(word, letter_bank):
-    pass
+    # the purpose of this function is to check if the letters in the user input
+    # are in present in our current hand.
+
+    # access our hand
+    list_copy = copy.deepcopy(letter_bank)
```

Comment:
[nit] Because `letter_bank` is a list of strings, it's sufficient to shallow copy it instead of deep copy it.

```python
    list_copy = letter_bank.copy()
```

Reply:
Also `list_copy` isn't a super descriptive name. A copy of which list? You could call it `letter_bank_copy`, or, since you don't need to refer to the original list that came in with the `letter_bank` parameter, you can just reassign it! 😲 

```python
    letter_bank = letter_bank.copy()
```